wsdl/ver10/schema/metadatastream.xsd

```diff
@@ -282,6 +282,11 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			</xs:element>
 			<xs:element name="Extension" type="tt:BehaviourExtension" minOccurs="0"/>
 			<xs:element name="Speed" type="xs:float" minOccurs="0"/>
+			<xs:element name="Direction" type="tt:GeoOrientation" minOccurs="0">
+				<xs:annotation>
```

Comment:
do we need to update onvif analytic service spec document to describe Direction parameter?

Reply:
Yes, good idea. I'll make an update.